<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Praca inżynierska</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addVerticeButton">
       <property name="toolTip">
        <string>TO są wielkie cycki hehehehehehheheeeeeeeeee noj lunukn y bny b yvb vb b y bhn n uk u n ny b bt v re r dx vgb yn
 uo njm ynbt  cetd  bn i mln bvf r cvu j</string>
       </property>
       <property name="text">
        <string>Dodaj wierzchołek</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="GraphView" name="graphView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>dateEdit</zorder>
   <zorder>graphView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuUstawienia">
    <property name="title">
     <string>Grafy</string>
    </property>
    <addaction name="actionShape"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuUstawienia"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddVertex"/>
   <addaction name="separator"/>
   <addaction name="actionAddEdge"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>O programie...</string>
   </property>
  </action>
  <action name="actionAddVertex">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/AddVerticeIcon.svg</normaloff>:/MainWindow/AddVerticeIcon.svg</iconset>
   </property>
   <property name="text">
    <string>Wierzchołek</string>
   </property>
   <property name="toolTip">
    <string>Dodaje nowy wierzchołek do grafu</string>
   </property>
  </action>
  <action name="actionShape">
   <property name="text">
    <string>Kształt...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+K</string>
   </property>
  </action>
  <action name="actionAddEdge">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/AddEdgeIcon.svg</normaloff>:/MainWindow/AddEdgeIcon.svg</iconset>
   </property>
   <property name="text">
    <string>Krawędź</string>
   </property>
   <property name="toolTip">
    <string>Dodaje krawędź pomiędzy wierzchołkami</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QGraphicsView</extends>
   <header>GraphView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
